src/types/questionnaire/form.ts

```diff
@@ -17,9 +17,8 @@ type RV<T extends string, V> = {
   coding?: Code;
   unit?: Code;
   type: T;
-  value?: V;
+  value: V | Code | number;
```

Comment:
V is a generic type! Pass V as RV's generic's arguments

```suggestion
  value: V;
```